samcli/lib/pipeline/init/questions.json

```diff
@@ -0,0 +1,182 @@
+{
+  "questions": [{
+    "key": "git_provider_type",
+    "question": "What is the Git provider?",
+    "options": ["Bitbucket", "CodeCommit", "GitHub", "GitHubEnterpriseServer"],
+    "nextQuestion": {
+      "CodeCommit": "codecommit_repository_name"
+    },
+    "defaultNextQuestion": "codestar_full_repository_id"
+  }, {
+    "key": "codestar_full_repository_id",
+    "question": "What is the full repository id (Example: some-user/my-repo)?",
+    "isRequired": true,
+    "defaultNextQuestion": "main_git_branch"
+  }, {
+    "key": "codecommit_repository_name",
+    "question": "What is the CodeCommit repository name?",
+    "isRequired": true
```

Comment:
what is this new file for?

Reply:
Good catch, I added it for testing and forgot to remove.